test/util/utils.sh

```diff
@@ -27,3 +27,70 @@ function kubeconfig_cleanup {
     kubectl config delete-user "${cluster_name}-admin" || true
     kubectl config delete-cluster "${cluster_name}" || true
 }
+
+function collect_standalone_cluster_diagnostics {
```

Comment:
When any issue occurs during collection of diagnostics data - do we still go ahead and cleanup the cluster? This would mean that we might not be able to debug the failure that happened. Or should we leave the cleanup of the cluster as is so that we can debug and get diagnostics data and then clean up the cluster? As of now, it does not cleanup the cluster. For example - https://github.com/karuppiah7890/community-edition/runs/4189838503?check_suite_focus=true#step:5:400 it abruptly stops when there is an issue (in this case bug in script) in collecting diagnostics.

Reply:
If this script is being used for workflows (which I believe it is), we should probably still clean up the environment in order to prevent orphaned clusters requiring manual intervention to clean and also possibly these stacking up an eating resource which (depending on the environment) might cause other failures due to lack of resources.

Reply:
Cool ! Then I can swallow any errors in command using `|| true`